sections/faqs/browser-support.js

```diff
@@ -0,0 +1,11 @@
+import md from "components/md"
+
+const BrowserSupport = () => md`
+  ## Which browsers are supported?
+
+  All [browsers that React supports](https://reactjs.org/docs/react-dom.html#browser-support) are supported.
+
+  All major browsers including Internet Explorer 9 and above are supported.
```

Comment:
Terribly sorry to do this to you 😢 but I didn't realise that it'll look and sound repetitive when I suggested it. Maybe we can do:

> This means all major browsers including Internet Explorer 9.

Either way though, this seems good to go :+1: Thanks for the contribution

Reply:
No problem at all. 
I appreciate you taking the time to provide some guidance on this. 😃